src/SpecDataHolder.sol

```diff
@@ -90,17 +136,6 @@ contract SpecDataHolder is
     return raftOwner == _admin || raft.isAdminActive(_raftTokenId, _admin);
   }
 
-  // Passing in the owner's address allows an EOA to deploy and set a multi-sig as the owner.
-  function initialize(address _raftAddress, address _nextOwner)
```

Comment:
I accidentally left a comment on the commit. It is unclear to me why we'd be deleting this? Could you provide context, please?

Reply:
it's not deleted, it's just moved up. It's standard convention for the `initialize` function to be after the constructor (not sure why it's been at the bottom all this time)